src/osgPlugins/OpenFlight/expGeometryRecords.cpp

```diff
@@ -50,7 +50,7 @@ static unsigned int LAYER_7( 0x80000000 >> 6 );
 
 
 bool
-FltExportVisitor::isLit( const osg::Geometry& /*geom*/ ) const
```

Comment:
I think we want the 3.6 version of this file. There are no changes in 3.4 makOsg

Reply:
Reverted changes to this file (pending)